app/src/main/java/org/fossasia/pslab/fragment/LALogicLinesFragment.java

```diff
@@ -46,6 +49,27 @@ public static LALogicLinesFragment newInstance(Bundle params, Context context) {
     public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         this.channelMode = params.getInt("channelMode");
+        switch (channelMode) {
+            case 1:
+                channelNames.add(params.getString("inputChannel1"));
+                break;
+            case 2:
+                channelNames.add(params.getString("inputChannel1"));
+                channelNames.add(params.getString("inputChannel2"));
+                break;
+            case 3:
+                channelNames.add(params.getString("inputChannel1"));
+                channelNames.add(params.getString("inputChannel2"));
+                channelNames.add(params.getString("inputChannel3"));
+                break;
+            case 4:
+                channelNames.add(params.getString("inputChannel1"));
+                channelNames.add(params.getString("inputChannel2"));
+                channelNames.add(params.getString("inputChannel3"));
+                channelNames.add(params.getString("inputChannel4"));
+                break;
+            default:
```

Comment:
This can be simplified with removing break and arranging this in reverse order! :) Have a look at this [link](https://stackoverflow.com/questions/8563970/switch-without-break)